oauth2-authorization-server/src/main/java/org/springframework/security/config/annotation/web/configurers/oauth2/server/authorization/OAuth2TokenRevocationEndpointConfigurer.java

```diff
@@ -0,0 +1,65 @@
+package org.springframework.security.config.annotation.web.configurers.oauth2.server.authorization;
+
+import org.springframework.http.HttpMethod;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.config.annotation.ObjectPostProcessor;
+import org.springframework.security.config.annotation.web.HttpSecurityBuilder;
+import org.springframework.security.oauth2.server.authorization.config.ProviderSettings;
+import org.springframework.security.oauth2.server.authorization.web.OAuth2TokenRevocationEndpointFilter;
+import org.springframework.security.web.access.intercept.FilterSecurityInterceptor;
+import org.springframework.security.web.authentication.AuthenticationFailureHandler;
+import org.springframework.security.web.authentication.AuthenticationSuccessHandler;
+import org.springframework.security.web.util.matcher.AntPathRequestMatcher;
+import org.springframework.security.web.util.matcher.RequestMatcher;
+
+public class OAuth2TokenRevocationEndpointConfigurer extends AbstractOAuth2Configurer{
+	private AuthenticationSuccessHandler tokenRevocationSuccessHandler;
+	private AuthenticationFailureHandler tokenRevocationFailureHandler;
+	private RequestMatcher requestMatcher;
+
+	OAuth2TokenRevocationEndpointConfigurer(ObjectPostProcessor<Object> objectPostProcessor) {
+		super(objectPostProcessor);
+	}
+
+	public OAuth2TokenRevocationEndpointConfigurer tokenRevocationSuccessHandler(AuthenticationSuccessHandler tokenRevocationSuccessHandler){
```

Comment:
Please review the naming conventions in the other `AbstractOAuth2Configurer` implementations to ensure consistency.

Reply:
I have changed method names to `onAuthenticationSuccess` and `onAuthenticationFailure`